srtcore/epoll.cpp

```diff
@@ -808,6 +808,12 @@ int CEPoll::swait(CEPollDesc& d, map<SRTSOCKET, int>& st, int64_t msTimeOut, boo
     return 0;
 }
 
+bool CEPoll::empty(CEPollDesc& d)
```

Comment:
If `m_EPollLock` is marked as `mutable`, then:

```suggestion
bool CEPoll::empty(CEPollDesc& d) const
```